src/keys-builder/create-translation-files.ts

```diff
@@ -23,21 +24,24 @@ export async function createTranslationFiles({
     output,
     fileFormat,
   });
-  const globalFiles = langs.map((lang) => ({
-    path: `${output}/${lang}.${fileFormat}`,
-  }));
+
   const actions: FileAction[] = [];
 
-  for (const { path } of globalFiles) {
-    actions.push(
-      buildTranslationFile({
-        path,
-        translation: scopeToKeys.__global,
-        replace,
-        removeExtraKeys,
-        fileFormat,
-      }),
-    );
+  if (!scopedOnly) {
+    const globalFiles = langs.map((lang) => ({
+      path: `${output}/${lang}.${fileFormat}`,
+    }));
+    for (const { path } of globalFiles) {
```

Comment:
Generally speaking, shouldn't we just skip creation if there aren't any keys there? instead of passing it as a property